logging/interceptor.go

```diff
@@ -109,12 +135,42 @@ func UnaryServerInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnarySer
 		levelLogf(
 			ctxlogrus.Extract(newCtx).WithFields(fields),
 			options.codeToLevel(code),
-			"finished unary call with code "+code.String())
+			fmt.Sprintf("finished unary call with code %s", code.String()))
 
 		return resp, err
 	}
 }
 
+func StreamServerInterceptor(entry *logrus.Entry, opts ...Option) grpc.StreamServerInterceptor {
+	options := initOptions(opts)
+
+	return func(srv interface{}, stream grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+		startTime := time.Now()
+		fields := newLoggerFields(info.FullMethod, startTime, DefaultServerKindValue)
+		newCtx := newLoggerForCall(stream.Context(), entry, fields)
+
+		newCtx = setInterceptorFields(newCtx, fields, entry.Logger, options, startTime)
+
+		wrapped := grpc_middleware.WrapServerStream(stream)
+		wrapped.WrappedContext = newCtx
+
+		err := handler(srv, wrapped)
+		if err != nil {
+			fields[logrus.ErrorKey] = err
+		}
+
+		code := status.Code(err)
+		fields[DefaultGRPCCodeKey] = code.String()
+
+		levelLogf(
+			ctxlogrus.Extract(newCtx).WithFields(fields),
+			options.codeToLevel(code),
+			fmt.Sprintf("finished server streaming call with code %s", code.String()))
```

Comment:
Per definition of levelLogf ```func levelLogf(entry *logrus.Entry, level logrus.Level, format string, args ...interface{}) {``` we can add arguments directly into this function instead of using fmt.Sprintf